### DISCUSSION TITLE
Possible Bug? WP Page name being replaced by Meeting name in List view

### DISCUSSION THREAD

**Discussion by steve970** [code4recovery/12-step-meeting-list#1579] [Q&A]
Hello,

I noticed something the other day that I am trying to understand. In my header, I have a div with a class called inner_title. The inner_title is usually the WP Page name across every single page on my site, except for the page with the 12-Step-Meeting-List Plugin List view. In this case, the inner_title is the first meeting on the list. When I click into an individual meeting, the inner_title changes to the appropriate meeting name, as expected. 

The expected outcome when you click on my WP Page name Meetings and see the list view is that the inner_title would also be Meetings and not the name of the first meeting on the list. Any Thoughts?

**Comment by gkovats** [comment]
Hi @steve970 can you share a URL for the issue you're seeing?

**Comment by steve970** [comment]
It's hard to share a URL since I am on a local dev. environment.

**Reply by steve970** [reply]
Happy to provide screenshots, otherwise, I can update this thread once I get it out of testing.

**Reply by gkovats** [reply]
I gotcha. You know if you're using Legacy UI or TSML UI?
![image](https://github.com/user-attachments/assets/043c261c-51f1-4e45-a8dc-e3e155d2acc4)
I question if there's something unique about your theme. I don't see the same behavior with the page title.
Screenshot wouldn't hurt either.
